### ISSUE TITLE
Can't Ctrl+drag to Exit

### ISSUE THREAD

**Issue by ivdma** [rubymotion/ib#82]
ib generates IBActions as follows: `- (IBAction)someMethodName:(id) sender;`, but Interface Builder's Exit expects it to be `- (IBAction)someMethodName:(UIStoryBoard *) sender;` in order it to be draggable to Exit.

I ended up editing stubs.h manually, but it would be nice to have it done by ib gem.

**Comment by yury** [comment]
@ivdma 

Can you check if `- (IBAction)someMethodName:(UIStoryBoard *) storyBoard;` also work? 
If so, we can check for parameter name.

**Comment by ivdma** [comment]
@yury 

Yes, `storyBoard` parameter works too.

**Comment by colinta** [comment]
Please test this new feature!  Available in 0.8.0.  Thanks, @yury!

**Comment by KCErb** [comment]
Hey there,

Three things:

1. You have a little typo: it should be `UIStoryboard` not `UIStoryBoard` ([source](https://developer.apple.com/library/ios/documentation/UIKit/Reference/UIStoryboard_Class/)).

2. I noticed the above because Xcode was giving me an error, but even after I fixed it I still had to change the type to `UIStoryboardSegue` in order to hook up the exit button! I'm confused because others seem to not have run into these problems. Suggestions?


3. Am I right to say that this is a totally new feature? It doesn't seem like you do parameter name checking anywhere else, but I'm not confident about that. 

To be clear to anyone else coming here to solve this problem: in order to get `ib` to give a `UIStoryboard` type you have to use the keyword `storyboard` or `story_board` as a parameter name in your method definition like so:

```ruby
def exit_method_name(storyboard)
end
```

and that will produce

```objc
-(IBAction) unwind_to_list:(UIStoryBoard *) storyboard;
```
